thunder/images/images.py

```diff
@@ -79,35 +80,44 @@ def toblocks(self, size='150', padding=None):
         from thunder.blocks.blocks import Blocks
         from thunder.blocks.local import LocalChunks
 
+        if chunk_size is 'auto':
+            chunk_size = str(int(1000000.0/self.shape[0]))
```

Comment:
Ah ah, ok great, just had to think through it, yeah that totally makes sense. Shouldn't the equation take `dtype` into account though?

And before merging, let's do some sanity sets on real, large datasets to make sure the numbers come out as expected. Maybe you've already done that.